index.html

```diff
@@ -176,6 +176,10 @@ <h1>The Registration Process</h1>
 The JSON-LD Context MUST be included in full as part of the submission.
           </li>
           <li>
+A cryptographic hash of the JSON-LD Context MUST be included as part of the
```

Comment:
```suggestion
A cryptographic hash of the JSON-LD Context SHOULD be included as part of the
```

Reply:
Language in DID Core is not a MUST, "These URIs SHOULD be associated with a cryptographic hash of the content of the JSON-LD Context as part of registration in the DID Specification Registries"

Reply:
Language in DID Core can also apply to things not in the Registries. We could use stronger language. But if there are genuine reasons for not doing this (other than "it's too hard work") the SHOULD makes sense.